t/plugin/jwt-auth.t

```diff
@@ -1140,3 +1140,48 @@ base64_secret required but the secret is not in base64 format
 [error]
 --- request
 GET /t
+
```

Comment:
@membphis does this test case cover the situation you mentioned
https://github.com/apache/apisix/pull/2657#discussion_r519373662